release/release.go

```diff
@@ -1066,10 +1066,15 @@ func uploadRelease(v version.Version) {
 	}
 }
 
-var linuxRepoVersions = []string{
-	"4.0.0-rc0", // any rc version will send the package to the "testing" repo
-	"4.3.0",     // any 4.3 stable release version will send the package to the "4.3" repo
-	"4.4.0",     // any 4.4 stable release version will send the package to the "4.4" repo
+var linuxRepoVersionsStable = map[string]string{
+	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
```

Comment:
I don't understand why these versions are hardcoded. How does this work?

Reply:
`curator` and `barque` bake in the assumption that they can just look at a MongoDB version and decide what linux repo the release is supposed to go to. The tools, however, are not on the same versioning scheme as the tools, so we can't just pass along our version directly (otherwise all of our GA releases would go to non-existent `100.X` repos).

This map contains a mapping of the actual repo names to a MongoDB version string that will cause `curator` and `barque` to send the release to that repo. The specific version strings aren't important, just some details about them (which I've tried to explain in the comments). We use this in the loop below, so that we can log that we're releasing to `ubuntu2004-org-testing`, but pass `--version 4.0.0-rc0` to curator so it goes to `testing`

Reply:
Ah I see!